bundle/android/strings.xml

```diff
@@ -91,6 +91,7 @@
     <string name="gutenberg_native_choose_images" tools:ignore="UnusedResources">Choose images</string>
     <string name="gutenberg_native_choose_video" tools:ignore="UnusedResources">Choose video</string>
     <string name="gutenberg_native_clear_search" tools:ignore="UnusedResources">Clear search</string>
+    <string name="gutenberg_native_clear_selected_color" tools:ignore="UnusedResources">Clear selected color</string>
```

Comment:
This new entry was added in with [this recent change](https://github.com/WordPress/gutenberg/pull/57650/files#diff-9f6b7d8a4fbcc5df8f8da7e9b3c0d4dc05bac08d5232dce16dee260a2ba8cdc8R116).